scripts/setup.sh

```diff
@@ -186,6 +186,9 @@ if [ ! -d "$NODE_PATH" ]; then
   chmod -R 744 $NODE_MODULE_DIR
 fi
 
+# Change execute status of browsermob-proxy
+chmod +x $TOOLS_DIR/browsermob-proxy-2.1.1/bin/browsermob-proxy
```

Comment:
Also, do we actually need this? Sean pointed out there is a node/npm module for browsermob-proxy (https://www.npmjs.com/package/browsermob-proxy). Does that case need the executable bit flipped?